### PR TITLE
chore(unitt):fix the unit test of rest service with maven-invoker

### PR DESCRIPTION
**Pull request by sgandon**
**What is the new behavior?**
Rest service unit test for integration shall pass now. They are launche using maven-invoker after a local maven repo is created

**What is the current behavior?** (You can also link to an open issue here)
rest serrvice test are failing

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/25/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/25/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/32/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/32/)

File: services/components-api-service-rest/src/it/jdbc/pom.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		</pluginRepository>
+	</pluginRepositories>
+
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.12.4</version>
+				<configuration>
+					<argLine>-Dorg.ops4j.pax.url.mvn.localRepository=/Users/sgandon/Talend/git/components/services/components-api-service-rest/target/local-repo</argLine>
```
Comment (gonchar-ivan) [inline services/components-api-service-rest/src/it/jdbc/pom.xml]: This configuration looks strange.
There is your username in path (sgandon)
Path to repo seems to be incorrect

File: services/components-api-service-rest/pom.xml [hunk 2]
```diff
@@ -199,6 +199,41 @@
                     <skipEnunciate>true</skipEnunciate>
                 </configuration>
             </plugin>
+			<plugin>
+				<artifactId>maven-invoker-plugin</artifactId>
```
Comment (gonchar-ivan) [inline services/components-api-service-rest/pom.xml]: Following link may be useful to understand how this plugin is used:
http://maven.apache.org/plugins/maven-invoker-plugin/usage.html

Comment (sgandon) [inline services/components-api-service-rest/pom.xml]: I will add some explanation

File: services/components-api-service-rest/src/it/jdbc/pom.xml (hunk 1 above)
Comment (sgandon) [inline services/components-api-service-rest/src/it/jdbc/pom.xml]: I removed it completely it was not used.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/45/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/45/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/46/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/46/)

**Pull request merged**